packages/zendesk-adapter/src/filters/unordered_lists.ts

```diff
@@ -125,7 +125,8 @@ const isValidConditions = (conditions: unknown, customFieldById: Record<string,
       isReferenceExpression(condition.value)
       && (customFieldById[condition.value.elemID.getFullName()]?.value.value !== undefined)
     )
-    || (isString(condition.value)))
+    || (isString(condition.value))
+|| (isBoolean(condition.value)))
```

Comment:
1. nit:
```suggestion
    || _.isString(condition.value)
    || _.isBoolean(condition.value))
```
2. do we know these are all the possible values? could it be e.g. a number as well?